Source/Manager/User/User.swift

```diff
@@ -251,17 +258,23 @@ public class User: UserProtocol {
         self.isPersistent = true
     }
 
-    func updateStoredTokens(_ tokens: TokenData?, previousTokens: TokenData?) {
-        // Always clear old tokens
+    private static func updatePersistedTokens(_ tokens: TokenData?, previousTokens: TokenData?) {
```

Comment:
Since the two operations are now decoupled, could it be clearer to separate this into two methods? (One for storing tokens and another one for clearing).